pages/common/safe.md

```diff
@@ -0,0 +1,36 @@
+# safe
+
+> A CLI to interact with HashiCorp Vault.
+> More information: <https://github.com/starkandwayne/safe>.
+
+- Add a safe target:
+
+`safe target {{vault_addr}} {{target_name}}`
+
+- Authenticate the CLI client against the Vault server, using an authentication token:
+
+`safe auth {{authentication_token}}`
+
+- Print the environment variables describing the current target:
+
+`safe env`
+
+- Provide a tree hierarchy listing of all reachable keys in the Vault:
+
+`safe tree {{path}}`
+
+- Move a secret from one path to another:
+
+`safe move {{old_path/secret}} {{new_path/secret}}`
```

Comment:
I don't like these parameter names very much. I would prefer if we could use the default style like `path/to/secret` but I don't know how to rewrite this properly so I think that's good haha.

Reply:
@schneiderl Thanks for the review. Maybe to put `old/path/secret` `new/path/secret`. Not sure  🤔